debian/changelog

```diff
@@ -1,3 +1,33 @@
+ubuntu-advantage-tools (27.10~23.04.1) kinetic; urgency=medium
+
+  * d/control:
+    - Add ca-certificates dependency
+  * d/rules:
+    - Only install APT hooks on LTS series
+  * New upstream release 27.10 (LP: #1980990)
+    - api: add support for new api command for version and magic-attach
+    - apt-hook:
+      + update json hook messages
+      + replace golang with cpp for json-hook
+    - cli
+      + properly sort services for detach/attach (GH: #1831)
+      + collect-logs include rotated log files
+      + display UA features directly on status
+      + Add support for pro command
+    - esm: Rename ESM service title and description
+    - esm-apps: make service non-beta
+    - fix:
+      + update ua portal url when asking for attach
+      + add --dry-run option
+    - gcp-pro: handle metadata endpoint error
+    - messages:
+      + remove warranty message from MOTD and APT
+      + update base UA url on all messages
+    - timer: log when job start running
+    - security-status: include download size of package updates
```

Comment:
We're missing a line for not starting the daemon during auto attach which fixes LP: #1980865